aptos-move/framework/aptos-framework/sources/transaction_validation.spec.move

```diff
@@ -0,0 +1,116 @@
+spec aptos_framework::transaction_validation {
+    spec module {
+        pragma verify = true;
+        // pragma aborts_if_is_strict;
```

Comment:
Why is this commented out?

Reply:
Because the aborts_if in the functions multi_agent_script_prologue and epilogue are more complicated, aborts_if_is_strict is annotated to ensure that the verification can pass. Now aborts_if_is_strict has been added, and aborts_if_is_partial is used in these two functions.

Reply:
Because the aborts_if in the functions `multi_agent_script_prologue` and `epilogue` are more complicated, aborts_if_is_strict is annotated to ensure that the verification can pass. Now aborts_if_is_strict has been added, and aborts_if_is_partial is used in these two functions.